components/style/gecko_bindings/check_bindings.py

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import re
+
+bindings = open("bindings.rs", "r")
+tests = open("test_bindings.rs", "w")
+
+tests.write("/* This Source Code Form is subject to the terms of the Mozilla Public\n")
+tests.write(" * License, v. 2.0. If a copy of the MPL was not distributed with this\n")
+tests.write(" * file, You can obtain one at http://mozilla.org/MPL/2.0/. */\n\n")
```

Comment:
You did the hooking up to regen correctly 😄 

As I said elsewhere, don't worry about including it as a mod just yet.